maintainer/CI/build_cmake.sh

```diff
@@ -130,7 +130,10 @@ if [ "${with_coverage}" = true ]; then
 fi
 
 cmake_params="-DCMAKE_BUILD_TYPE=${build_type} -DWARNINGS_ARE_ERRORS=ON -DTEST_NP:INT=${check_procs} ${cmake_params}"
-cmake_params="${cmake_params} -DCMAKE_CXX_FLAGS=${cxx_flags} -DCUDA_NVCC_FLAGS=${nvcc_flags}"
+cmake_params="${cmake_params} -DCMAKE_CXX_FLAGS=${cxx_flags}"
```

Comment:
why do we manually set cxx flags here?

Reply:
Maybe we can replace that logic with the corresponding `-DCMAKE_BUILD_TYPE=Debug`. This is again overlapping with the next PR, and might also depend on #2977:
https://github.com/espressomd/espresso/blob/be99119c8113b1f2a40ca51d5b07b738bf981eee/.gitlab-ci.yml#L407

Reply:
all right, let's do it in the other PR